SchoolofComputerScience/scs-app
# Adding API Call for Scid before going to Member Pages

# Description
Some scids, are people who are not found in out directory collection (which is constructed from WorkDay). One example of this, is the instructor of some courses. A check to validate the scid is now in place before going to Member pages for a scid that is not found in our directory.

# Test Instructions
1. Go to `/courses/course/M17-05589-A`
2. Click on the instructor
3. Notice that a pop-up notification will be displayed and you will not be taken to an erroneous Member page
